README.rst

```diff
@@ -47,9 +47,14 @@ names by configuring ``DISCOVERY_INDEX_OVERRIDES``::
 Re-indexing courses
 ~~~~~~~~~~~~~~~~~~~
 
-::
+While using tutor in development mode, use ``--partner_code=dev`` and in production mode, use ``--partner_code=openedx`` in refresh_course_metadata command as mentioned below::
 
+    tutor dev run discovery ./manage.py refresh_course_metadata --partner_code=dev
+    or
```

Comment:
Instead of or, we can have two code blocks, one to specify dev and one for local.

Reply:
I have separated the blocks, can u look at it again?

Reply:
We should probably make a similar change here as well:
https://github.com/overhangio/tutor-discovery/blob/a40d767426918edbf41c11957475b838c4a40975/README.rst?plain=1#L68-L72

Reply:
Done some changes here as well.